compiler/resolution/callDestructors.cpp

```diff
@@ -590,7 +590,20 @@ void ReturnByRef::transformMove(CallExpr* moveExpr)
                 isSyncType(formalType) == false &&
                 isSingleType(formalType) == false)
             {
-              copyExpr = rhsCall;
+              // In functionResolution, insertAutoCopyForArrayReturn inserts
+              // AST like this:
+              //
+              // (move ret chpl__autoCopy(foo))
+              //
+              // We do not want to remove this autoCopy as it implements the
+              // desired copy-out semantics of the language. The 'ret' variable
+              // should be marked with FLAG_RVV, so we can test for that case
+              // here and avoid it.
```

Comment:
I would expect this to do etiher/both of:
 * check for Array View type
 * add a call to a fn with a different name in function resolution (e.g. "chpl__unalias")